lib/pbxProject.js

```diff
@@ -286,9 +287,24 @@ pbxProject.prototype.removeFromPbxBuildFileSection = function (file) {
     }
 }
 
-pbxProject.prototype.addPbxGroup = function (filePathsArray, name, path, sourceTree) {
+pbxProject.prototype.findMainPbxGroup = function () {
+    var groups = this.hash.project.objects['PBXGroup'];
+    var candidates = [];
+    for (var key in groups) {
+        if (groups[key].path == undefined && groups[key].name == undefined && groups[key].isa) {
+            candidates.push(groups[key]);
+        }
+    }
+    if (candidates.length == 1) {
+        return candidates[0];
+    }
+
+    return null;
```

Comment:
We should have exactly one candidate as this will be the root group. If they are more than one there should be an error. Maybe we can throw an exception.